src/ActiveLogin.Authentication.BankId.Api/BankIdApiClientExtensions.cs

```diff
@@ -48,6 +48,58 @@ public static Task<AuthResponse> AuthAsync(this IBankIdApiClient apiClient, stri
             return apiClient.AuthAsync(new AuthRequest(endUserIp, personalIdentityNumber));
         }
 
+        /// <summary>
+        /// Initiates an authentication order. Use the collect method to query the status of the order.
+        /// </summary>
+        /// <param name="apiClient">The <see cref="IBankIdApiClient"/> instance.</param>
+        /// <param name="endUserIp">
+        /// The user IP address as seen by RP.String.IPv4 and IPv6 is allowed.
+        /// Note the importance of using the correct IP address.It must be the IP address representing the user agent (the end user device) as seen by the RP.
+        /// If there is a proxy for inbound traffic, special considerations may need to be taken to get the correct address.
+        ///
+        /// In some use cases the IP address is not available, for instance for voice based services.
+        /// In this case, the internal representation of those systems IP address is ok to use.
+        /// </param>
+        /// <param name="userVisibleData">
+        /// The text to be displayed and signed. String. The text can be formatted using CR, LF and CRLF for new lines. The text must be encoded as UTF-8 and then base 64 encoded. 1--40 000 characters after base 64 encoding.
```

Comment:
See comment below on Base64-handling. Therefore the XML doc should not mention the Base64 part. All XML-docs for `userVisibleData` should be updated to:
`The text to be displayed and signed. The text can be formatted using CR, LF and CRLF for new lines.`